readthedocs/projects/tasks.py

```diff
@@ -466,18 +472,20 @@ def run_build(self, docker, record):
                 self.update_documentation_type()
 
             python_env_cls = Virtualenv
-            if self.config.use_conda:
+            if self.config.conda:
```

Comment:
Conda is None now, if the user is using conda this is a dictionary

Reply:
Check against explicit `None` here.

    if self.config.conda is not None: